Gemfile.lock

```diff
@@ -301,7 +301,7 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (7.0.0)
       racc
-    browser (6.2.0)
+    browser (5.3.1)
```

Comment:
I am not sure about the gemfile changes here, I think they were just generated after running `bundle install` since merging in the ruby upgrade changes in release-153, but maybe they should be reverted, or some of these gems should actually be updated to later versions? To investigate more